docs/Matter_TH_User_Guide/Matter_TH_User_Guide.adoc

```diff
@@ -1114,12 +1117,12 @@ image:images/img_15.png[]
 [source,xml]
 ----
 "dut_config": {
-   "discriminator": "3840",
-   "setup_code": "20202021",
-   "pairing_mode": "ble-wifi",
-   "chip_timeout": null,
-   "chip_use_paa_certs": false,
-   "trace_log": true
+  "discriminator": "3840",
+  "setup_code": "20202021",
+  "pairing_mode": "ble-wifi",
+  "chip_timeout": null,
+  "chip_use_paa_certs": false,
+  "trace_log": true
 }
 ----
```

Comment:
To maintain the same order as the others pairing modes, I suggest to switch places between the items `a` and `b` of the **Wi-Fi Mode** section.

Reply:
Solved in https://github.com/project-chip/certification-tool/pull/405/commits/d6514cd7fddc91af92a74c9fc3f3ab6d642d7ce1